meta-luneos/recipes-connectivity/ofono/ofono_%.bbappend

```diff
@@ -1,37 +1,56 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 
-SRCREV = "717f6452aad0d15bf9b3af3491fdd86b87d31315"
-PV = "1.21+git${SRCPV}"
+SRCREV = "0db662bd6ba4070838bf143df5ee24c949a8c0df"
+PV = "1.31+git${SRCPV}"
+
+SRCREV_halium = "717f6452aad0d15bf9b3af3491fdd86b87d31315"
+PV_halium = "1.21+git${SRCPV}"
 
 LICENSE = "GPLv2"
 LIC_FILES_CHKSUM = "file://COPYING;md5=eb723b61539feef013de476e68b5c50a"
 
 DEPENDS += "libglibutil dbus-glib libgrilio libmce-glib"
 RDEPENDS_${PN} += "mobile-broadband-provider-info ofono-conf"
 
-SRC_URI  = " \
+SRC_URI = " \
+  git://git.kernel.org/pub/scm/network/ofono/ofono.git \
+  file://0001-common-create-GList-helper-ofono_call_compare.patch \
+  file://0002-common-atmodem-move-at_util_call_compare_by_status-t.patch \
+  file://0003-common-atmodem-move-at_util_call_compare_by_id-to-dr.patch \
+  file://0004-add-call-list-helper-to-manage-voice-call-lists.patch \
+  file://0005-qmimodem-implement-voice-calls.patch \
+  file://ofono \
+  file://ofono.service \
+"  
+
+SRC_URI_halium  = " \
   git://git.merproject.org/mer-core/ofono.git \
   file://0001-Enable-the-various-modem-plugins-we-support-again.patch;striplevel=2 \
   file://0002-Add-support-for-the-Ericsson-F5521gw-modem.patch;striplevel=2 \
   file://ofono \
-  file://ofono.service \
+  file://ofono-halium.service \
 "
 
-S = "${WORKDIR}/git/ofono"
+S = "${WORKDIR}/git"
+S_halium = "${WORKDIR}/git/ofono"
 
 # Can't build out of tree right now so we have to build in tree
 B = "${S}"
 
-EXTRA_OECONF_append = " --disable-sailfish-pushforwarder --enable-sailfish-rilmodem"
+EXTRA_OECONF_halium_append = " --disable-sailfish-pushforwarder --enable-sailfish-rilmodem"
```

Comment:
Is this really what you wanted to do?
EXTRA_OECONF_append_halium
makes much more sense and is equivalent of what was there before.